samples/custom-hostname-instance/getInstanceHostname.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Prints a hostname of the GCP VM instance.
+ *
+ * @param {string} projectId - ID or number of the project you want to list images from
+ * @param {string} zone - name of the zone your instance belongs to, for example: us-west3-b
+ * @param {string} instanceName - Name of the new machine.
+ */
+function main(projectId, zone, instanceName) {
+  // [START compute_instances_get_hostname]
+  /**
+   * TODO(developer): Uncomment and replace these variables before running the sample.
```

Comment:
To do (developer): Uncomment and specify values for the following variables before running the sample.